pandapower/converter/powermodels/to_pm.py

```diff
@@ -390,6 +392,8 @@ def ppc_to_pm(net, ppci):
         logger.warning("PowerModels.jl does not consider reactive power cost - costs are ignored")
         ppci["gencost"] = ppci["gencost"][:ppci["gen"].shape[0], :]
     for idx, row in enumerate(ppci["gencost"], start=1):
```

Comment:
iterating through gencost is very slow for grids with many generators. can you vectorize?